master/geometry.html

```diff
@@ -32,7 +32,7 @@ <h2 id='CX'>Horizontal center coordinate: The
   </tr>
   <tr>
     <th>Value:</th>
-    <td><a>&lt;length&gt;</a> | <a>&lt;percentage&gt;</a></td>
+    <td><a>&lt;length-percentage&gt;</a></td>
```

Comment:
These behave different to properties like width. At least in Safari/Webkit (only browsers to support them), a percentage value doesn't compute to a px value and calc(50% + 10px) for example does work but the computed value stays as  calc(50% + 10px). I'm not sure if that is a bug or not. 

Does this meet the criteria: "Equivalent to `[ <length> | <percentage> ],` where the `<percentage> `will resolve to a `<length>`." Example: "the width property can accept a `<length>` or a `<percentage>`, both representing a measure of distance. This means that `width: calc(500px + 50%);` is allowed—both values are converted to absolute lengths and added."

Reply:
The browser implementations are bad, then. They likely have not updated their list of "applies to" elements (that is, elements for which `getComputedStyle` is supposed to return the used value instead of the inherited "computed" value).

Percentages in these properties describe lengths, they should compute to lengths, and they should be combinable with lengths in `calc`.